lib/Extension/LanguageServerCodeTransform/Tests/Unit/CodeAction/PropertyAccessGeneratorProviderTest.php

```diff
@@ -17,7 +16,8 @@
 use Prophecy\PhpUnit\ProphecyTrait;
 use function Amp\Promise\wait;
 
-class GenerateAccessorProviderTest extends TestCase
+/** @group wip */
+class PropertyAccessGeneratorProviderTest extends TestCase
```

Comment:
remove `@group` (btw can easily run a specific test by specifying `phpunit path/to/testTest.php --filter="provide actions`)

Reply:
:see_no_evil: whoops